changelog/2241.changed.md

```diff
@@ -0,0 +1 @@
+`ClockId::set_time()` has been disabled on iOS/tvOS/watchOS given that it is only available on macOS
```

Comment:
It was already disabled on iOS/tvOS/watchOS.  Shouldn't this read "ClockId::set_time is now enabled on macos" ?